autorest/codegen/models/code_model.py

```diff
@@ -368,3 +368,11 @@ def generate_single_parameter_from_multiple_media_types(self) -> None:
                         raise ValueError("You are missing a parameter that has multiple media types")
                     chosen_parameter.multiple_media_types_type_annot = f"Union[{type_annot}]"
                     chosen_parameter.multiple_media_types_docstring_type = docstring_type
+
+    @property
```

Comment:
ideally would've had this code in jinja2, but spent a long time and don't think I can get the jinja2 template to correctly filter by property 'has_lro'. Always defaults to what an Operation model has for this value: False